sdv/multi_table/utils.py

```diff
@@ -429,16 +430,29 @@ def _get_rows_to_drop(data, metadata):
     return table_to_idx_to_drop
 
 
+def _get_idx_to_drop_nan_foreign_key_table(data, relationships, table):
```

Comment:
minor: I'd rename this to something like `_get_nan_fk_indices_table` or something like that. The indexes aren't dropped here and if this method is useful elsewhere it might not be to ddrop the indices so I'd leave that part out